prodavnica/src/main/java/com/levi9/prodavnica/repository/BookRepository.java

```diff
@@ -14,9 +14,11 @@
 @Repository
 public interface BookRepository extends JpaRepository<Book, Long> {
 
-    @Query(value = "select b.book_id from book as b right join book_category  as bc on b.book_id = bc.book_id where bc.category_id=?1",nativeQuery = true)
-    List<Long> getBooksFromCategory(Long id);
-    
+    @Query(value = "SELECT * FROM book as b left join book_author as ba on b.book_id=ba.book_id left join author a" +
```

Comment:
Try with JPQL query. nativeQuery should be, generally avoided if there is JPQL alternative